src/main/java/com/iota/iri/controllers/TagViewModel.java

```diff
@@ -28,8 +27,16 @@ private TagViewModel(Tag hashes, Indexable hash) {
         this.hash = hash;
     }
 
+    public static TagViewModel load(Tangle tangle, Indexable hash, Class<?> model) throws Exception {
```

Comment:
why do we need `model`? 
Will it not always be `Tag.class`?

Reply:
see `loadObsolete()` - loading is done with both `Tag` and `ObsoleteTag` model.
so to not repeat the intialization code.

I can either make this `private` to be more explicit or just copy the `new` call to each.